src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -796,30 +802,35 @@ def task_stage_context(stage: Stage):
 
 
 @contextlib.contextmanager
```

Comment:
Could you add a comment describing why we need to receive the fuzz_target as well?